src/mumble/Cert.cpp

```diff
@@ -135,7 +135,7 @@ int CertWizard::nextId() const {
 					return 2;
 				else if (qrbExport->isChecked())
 					return 3;
-			}
+			} // Intentional fallthrough
```

Comment:
Please just add `return -1;` after the `if` block, to make the compiler happy instead.